open-vscode-dev.bat

```diff
@@ -0,0 +1 @@
+pixi run --environment=dev code . | exit
```

Comment:
Still opens vs-code in dev-environment:
- If I open a new interactive console it uses the default environment
- with `ctrl` + `shift` + `P` -> `Python: select Interpreter` -> `Enter interpreter path...` -> `my\path\to\Ribasim-NL\.pixi\envs\dev\python.exe` It won't be added to my envs list.

![image](https://github.com/Deltares/Ribasim-NL/assets/14051252/67d289ae-53ad-438e-b04e-6c64d52803ee)

Reply:
This should work better once VSCode recognizes pixi environments. Does this work for you? https://github.com/prefix-dev/pixi/issues/411#issuecomment-2107344663

Reply:
Reading that issue, you mean selecting the interpeter from the dev-env by this method: https://code.visualstudio.com/docs/python/environments#_manually-specify-an-interpreter.

Did some more checking; that doesn't work if I try to select the dev-env python.exe, but does work if I try to select the default-env python.exe

But I managed to find a workaround:
1. start a normal terminal `Terminal` -> `New Terminal` (or ctrl+shift+')
2. this dóes start in the dev-environment if you run vscode in the dev-environment
3. if you now open a interactive window, it starts to use the dev-environment as wel. And is also appears in the list of python environments to select from

In a screendump:
![image](https://github.com/Deltares/Ribasim-NL/assets/14051252/b7bf9004-1a66-4248-bcc5-aa1f2cab27ea)

Strange, but I think this is as good as it gets for now, so let's merge :-)

Reply:
That is curious, glad you found a workaround though.